.github/workflows/ci_cd.yml

```diff
@@ -52,7 +52,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ['3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
I would drop 3.9 as well. According to our guidance, it is no longer supported. Also, consider adding active support for 3.13 if possible. This would close #219

Reply:
Done.